src/libaktualizr/storage/invstorage.h

```diff
@@ -116,8 +116,8 @@ class INvStorage {
                                     InstalledVersionUpdateMode update_mode) = 0;
   virtual bool loadInstalledVersions(const std::string& ecu_serial, boost::optional<Uptane::Target>* current_version,
                                      boost::optional<Uptane::Target>* pending_version) const = 0;
-  virtual bool loadInstallationLog(const std::string& ecu_serial, std::vector<Uptane::Target>* log,
-                                   bool only_installed) const = 0;
+  virtual bool loadInstallationLog(const std::string& ecu_serial, std::vector<Uptane::Target>* log, bool only_installed,
+                                   bool allow_current = true) const = 0;
```

Comment:
I would break this change into two commits, one for the "allow_current" change and the second one for marking target as a "bad/failing" target.

Reply:
Probably we should call the variable as `include_current`.